### PR TITLE
Laboratory work #8, Matushchak Anastasia - 20FPL1

### PR DESCRIPTION
**Pull request by amatushchak**

### CODE REVIEW

**Review by SalnikovIgor — CHANGES_REQUESTED** [review]
Fix tests

File: requirements.txt [hunk 1]
```diff
@@ -0,0 +1,6 @@
+datasets==2.16.1
```
Comment (SalnikovIgor) [inline requirements.txt:1]: Add line `--extra-index-url https://download.pytorch.org/whl/cpu`

File: lab_7_llm/main.py [hunk 2]
```diff
@@ -1,11 +1,16 @@
 """
-Neural machine translation module.
+Neural summarization module.
```
Comment (SalnikovIgor) [inline lab_7_llm/main.py]: Checkout the files from 7 lab to the master

**Review by SalnikovIgor — CHANGES_REQUESTED** [review]

File: lab_7_llm/main.py (hunk 2 above)
Comment (amatushchak) [reply lab_7_llm/main.py]: done

File: requirements.txt (hunk 1 above)
Comment (amatushchak) [reply requirements.txt:1]: done

File: lab_8_llm/main.py [hunk 3]
```diff
@@ -4,28 +4,24 @@
 Working with Large Language Models.
 """
 # pylint: disable=too-few-public-methods, undefined-variable, too-many-arguments, super-init-not-called, duplicate-code
-from collections import namedtuple
+#from collections import namedtuple
```
Comment (SalnikovIgor) [inline lab_8_llm/main.py]: Why skip?

File: lab_8_llm/main.py [hunk 4]
```diff
@@ -34,6 +30,7 @@ class RawDataImporter(AbstractRawDataImporter):
     """
     A class that imports the HuggingFace dataset.
     """
+    _raw_data: DataFrame
```
Comment (SalnikovIgor) [inline lab_8_llm/main.py]: Delete and add in code a check: _raw_data containt None? If yes, then rise error

File: lab_8_llm/main.py [hunk 5]
```diff
@@ -44,6 +41,15 @@ def obtain(self) -> None:
             TypeError: In case of downloaded dataset is not pd.DataFrame
         """
 
+        self._raw_data = load_dataset(
+            path=self._hf_name,
+            split="train"
+        ).to_pandas()
+
+    @property
+    def raw_data(self) -> DataFrame:
```
Comment (SalnikovIgor) [inline lab_8_llm/main.py]: Also delete

**Review by SalnikovIgor — CHANGES_REQUESTED** [review]

File: lab_8_llm/main.py (hunk 3 above)
Comment (amatushchak) [reply lab_8_llm/main.py]: It is not used in code, so it should be deleted, I just forgor to delete it.

File: lab_8_llm/main.py (hunk 5 above)
Comment (amatushchak) [reply lab_8_llm/main.py]: Done

File: lab_8_llm/main.py (hunk 4 above)
Comment (amatushchak) [reply lab_8_llm/main.py]: Implemented a check

**Pull request closed without merging**